lib/index.ts

```diff
@@ -0,0 +1,173 @@
+/*
+	Copyright 2017 Balena Ltd.
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+*/
+
+import * as rSemver from 'resin-semver';
+import { actionsConfig as defaultActionsConfig } from './config';
+import { ActionName, ActionsConfig } from './types';
+export { actionsConfig } from './config';
+export * from './types';
+
+// ensure `version` is not a `dev` variant
+const isDevVariant = (version: string): boolean => {
+	const parsed = rSemver.parse(version);
+	if (parsed == null) {
+		return false;
+	}
+	return parsed.build.concat(parsed.prerelease).indexOf('dev') >= 0;
```

Comment:
```suggestion
	return parsed.build.concat(parsed.prerelease).includes('dev');
```

Reply:
The tsconfig is set to target es2015 (just like resin-semver), so this isn't available. Should I try a different target?
I was actually just trying to not include lodash as a dependency, but the SDK already does use it, so for now I will add lodash back as it was in the proxy.

Reply:
It's an es2016 lib feature but exists in every node version that supports es2015 from what I understand (ie node 6+)